### ISSUE TITLE
Lock rotation to single axis

### ISSUE THREAD

**Issue by nick-sp** [google/model-viewer#1240]
Is there a way to lock the user rotation of an object on a single axis (Y)?

**Comment by elalish** [comment]
Yes, by specifying the min/max camera orbit. `minCameraOrbit` has `theta, phi, radius`, where theta is around the vertical axis and phi is elevation. If you set min and max phi to the same value, then the user can't move it in that axis anymore.
